entity-framework/core/what-is-new/ef-core-6.0/breaking-changes.md

```diff
@@ -618,6 +618,27 @@ var hasDifferences = context.GetService<IMigrationsModelDiffer>().HasDifferences
     context.GetService<IDesignTimeModel>().Model.GetRelationalModel());
 ```
 
+This snippet shows how to implement <xref:Microsoft.EntityFrameworkCore.Design.IDesignTimeDbContextFactory%601> by creating a model externally and calling <xref:Microsoft.EntityFrameworkCore.DbContextOptionsBuilder.UseModel%2A>:
+
+```csharp
+internal class MyDesignContext : IDesignTimeDbContextFactory<MyContext>
```

Comment:
@AndriySvyryd Could we make this easier?

Reply:
It would be fairly involved and risky from our side, but we could do it if we get more reports.